extension.Rmd

```diff
@@ -0,0 +1,175 @@
+---
+title: "Gozde and Liora extension"
+author: "Liora and Gozde"
+date: "April 27, 2015"
+output:
+  html_document:
+    pandoc_args: [
+      "+RTS", "-K64m",
+      "-RTS"
+    ]
+---
+# EXTENSION
+
+Our extension was inspired by Andreas Wimmer and Yuval Feinstein's 2010 paper "The Rise of the Nation-State Across the World, 1816-2001."  This paper seperately tests the prevailing explanatory theories of the rise of the nation-state as the dominant form of governance. Our extension takes a similar approach, testing the major theories of support for extreme right parties in the Western European context in separate models. We operationalize the variables in these theories using the available measures from the original data set.
+
+In order to obtain a compelling visual representation of the predictive power of different theories and models, we use the method suggested in Brian Greenhill, Michael D. Ward and Audrey Sacks (2011) "The Separation Plot: A New Visual Method for Evaluating the Fit of Models with Categorical Outcomes",  American Journal of Political Science 55 (3): 1-13. We use their R package "separationplot".
+
+Run necessary packages and load data (Source: Arzheimer dataverse)
+
+```{r}
+
+
+library(MASS)
```

Comment:
sometimes I like to add a little comment next to certain packages about which function they afford, so that if I don't in the future need to use the function, I know which package I don't need. Loading unnecessary packages are inefficient and sometimes packages mask/change functions from other packages—i.e. it's not good practice